src/tools/pub/pubspec.md

```diff
@@ -34,6 +34,14 @@ A pubspec can have the following fields:
 : Optional.
   [_Learn more._](#homepage)
 
+`repository`
+: Optional. URL pointing to the repository.
```

Comment:
repository -> the package's source code

(or something else that doesn't just repeat "repository". we could say only "Optional. Learn more." like some other entries, but I like your idea of giving _some_ info here.)